src/Symfony/Component/Config/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * added option `ignore_not_found` for imported config files
```

Comment:
Should we add `ignore_not_found` (which allows passing both `ignore_not_found` and `ignore_errors` at the same time) ? Or should it look like `ignore_errors: 'not_found'` ?

Reply:
Your suggestion is brilliant. I have changed it to ignore_errors: 'not_found'. This it much better solution.